### PR TITLE
Admin

### LINKED ISSUE #6: Model the data
Going to use Postgres 9.4 for this.

### PR DESCRIPTION
**Pull request by mjtorn**
When this is merged, it closes #6

### CODE REVIEW

File: barbados/barbadosdb/admin.py [hunk 1]
```diff
@@ -1,3 +1,80 @@
...
+
+@admin.register(models.User)
+class CustomUserAdmin(UserAdmin):
+    """Django cannot find our custom user by default
+    """
+
+    fieldsets = (
+        (None, {'fields': ('username', 'password')}),
+        (_('Personal info'), {'fields': ('birth_date', 'first_name', 'last_name',
+                                        'email', 'phone_number',
```
Comment (greygriffin) [inline barbados/barbadosdb/admin.py]: Continuation line under-indented for visual indent

File: barbados/barbadosdb/admin.py [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+class BerthAdmin(admin.ModelAdmin):
+    """Deal with field order
+    """
+
+    # XXX: This is probably a database hammer
+    list_display = ('name', 'jetty', lambda self: self.jetty.harbour)
+
+    fieldsets = (
+        (None, {'fields':
+            ('name', 'jetty', 'boat'),
```
Comment (greygriffin) [inline barbados/barbadosdb/admin.py]: Continuation line under-indented for visual indent

File: barbados/barbadosdb/admin.py [hunk 3]
```diff
@@ -1,3 +1,80 @@
...
+    def edit_link(self):
+        """Name and registration can be empty but we need access
+        """
+
+        return '<a href={}>Edit</a>'.format(reverse('admin:barbadosdb_boat_change', args=(self.pk,)))
+    edit_link.allow_tags = True
+    edit_link.short_description = _('Edit link')
+
+    list_display = ('name', 'registration_number', 'sail_number', 'user', edit_link)
+
```
Comment (greygriffin) [inline barbados/barbadosdb/admin.py]: Blank line at end of file

File: barbados/barbadosdb/models.py [hunk 4]
```diff
@@ -235,3 +236,4 @@ class Meta:
 
     def __str__(self):
         return self.name
+
```
Comment (greygriffin) [inline barbados/barbadosdb/models.py]: Blank line at end of file

File: barbados/barbadosdb/admin.py [hunk 5]
```diff
@@ -1,3 +1,80 @@
...
+class BoatAdmin(admin.ModelAdmin):
+    """The thing that floats
+    """
+
+    def edit_link(self):
+        """Name and registration can be empty but we need access
+        """
+
+        return '<a href={}>Edit</a>'.format(reverse('admin:barbadosdb_boat_change', args=(self.pk,)))
+    edit_link.allow_tags = True
```
Comment (greygriffin) [inline barbados/barbadosdb/admin.py]: Somehow this bothers me, that the return statement is due to blank lines more closely associated with the following lines than with the def above. But it seems to be consistent with our formatting rules, so feel free to ignore this comment.

Comment (mjtorn) [inline barbados/barbadosdb/admin.py]: I usually have an empty line after docstrings so they don't look too crowded. The one with `allow_tags` is Django default formatting, which I hate, but I'm not sure really that it'd be better with a line apart...

**Comment by mjtorn** [comment]
Those flake8 errors above, they should all be ignored by the flake8 config, are you sure Sublime is getting it right? I double-checked that everything's clear here on my machine.

**Comment by greygriffin** [comment]
My Sublime setting was slightly wrong; now fixed.

**Pull request merged**